thesis-de/content-de.tex

```diff
@@ -327,6 +327,28 @@ \subsection{Datenerfassungssysteme}
 \subsection{Datenbanksystem / API}
 
 \subsection{Frontend}
+Die Weboberfläche besitzt zwei Schnittstellen, welche aufgeteilt sind, mit der
+Abfrage der Daten an dem Datenhaltungssystem und zum anderen mit der Ausgabe
+und Anzeige für den User bzw. dem Administrator. Eine Schnittstelle auf einem
+Webinterface dient in erster Linie zur Anzeige von Daten, die über ein anderes
+System(BackEnd) erhalten wurden. 
+
+Die erste Schnittstelle der Weboberfläche hat nur eine lesende Berechtigung,
```

Comment:
Die erste Schnittstelle der Weboberfläche ist ein standartisierter Treiber, welchen man verwenden kann um auf ein Datenhaltungssystem zuzugreifen. Dieser liefert der Weboberfläche alle nötigen Funktionen und Werkzeuge zur Datenextraktion, Datenmanipulation und der Rechteverwaltung auf einem Datenhaltungssystem. In der Ziellösung von diesem Projekt ist eine rein lesender Datenhaltungszugriff für die Weboberfläche vorgesehen. Dies wird von dem Datenhaltungssystem-Administrator gewährleistet.